### ISSUE TITLE
Noto Sans Mono font - Hebrew script is not mono

### ISSUE THREAD

**Issue by lyndondrake** [googlefonts/noto-fonts#2056]
The Noto Sans Mono font is great, but the Hebrew characters in it are not monospaced to match the Latin characters. This is easily verifiable by comparing the width of 10 yod characters:
יייייייייי
with the width of 10 mem characters:
ממממממממממ

**Comment by marekjez86** [comment]
@lyndondrake : that's correct. currently Noto supports only Mono Latin, Greek and Cyrillic characters

**Comment by lyndondrake** [comment]
Are there any plans for that to change? Or anything I can do to help that change take place?
